src/vs/base/browser/ui/sash/sash.ts

```diff
@@ -56,7 +58,14 @@ export const enum SashState {
 	Enabled
 }
 
+const _onDidChangeSize = new Emitter<number>();
+const onDidChangeSize: Event<number> = _onDidChangeSize.event;
+export function setSashSize(size: number): void {
+	_onDidChangeSize.fire(size);
+}
```

Comment:
Just a minor suggestion, let's maybe be a bit more explicit here and mention this is a global sash size:

```suggestion
const _onDidChangeGlobalSize = new Emitter<number>();
const onDidChangeGlobalSize: Event<number> = _onDidChangeGlobalSize.event;
export function setGlobalSashSize(size: number): void {
	_onDidChangeGlobalSize.fire(size);
}
```